docs/new-committer-checklist.md

```diff
@@ -0,0 +1,9 @@
+# New Committer Checklist
+
+- [ ] Announcement in General Meeting
+- [ ] Change in OWNERS.md
+- [ ] Access to Packet.net
+- [ ] Access to GCE
+- [ ] Access to Azure
+- [ ] Access to FOSSA
+- [ ] Access to Netlify
```

Comment:
https://github.com/networkservicemesh/networkservicemesh
https://github.com/networkservicemesh/site
https://github.com/orgs/networkservicemesh/people

Reply:
[ ] Access to twitter account
[ ] Access to docker hub account
[ ] Add as Admin to github org
[ ] Add as Admin to github repos

Reply:
I've added these, can you verify the changes are sufficient?

Reply:
👍